client/payment-details/timeline/test/map-events.js

```diff
@@ -12,6 +12,18 @@ jest.mock( 'gridicons/dist/checkmark', () => 'CheckmarkIcon' );
 jest.mock( 'gridicons/dist/cross', () => 'CrossIcon' );
 jest.mock( 'gridicons/dist/notice-outline', () => 'NoticeOutlineIcon' );
 
+// Mock the tax descriptions module
+jest.mock( '../../utils/tax-descriptions', () => ( {
```

Comment:
I was wondering why it's needed to mock this new file. Is it just for the `should handle invalid tax percentage rate with different currency` test case? Either way, this mocked `getLocalizedTaxDescription` seems to be implementing some logic (`mockTranslations[ description ] || description`) that isn't present in the main `client/payment-details/utils/tax-descriptions.ts` file. I wonder if this difference could become problematic.

Reply:
Good catch! I removed it in this commit [43c9d97](https://github.com/Automattic/woocommerce-payments/pull/10707/commits/43c9d9768378ba234a0d9b1c18a298082089646c). I think it made sense with my initial approach but I refactor things and forgot to update the tests 😅